jsonnet/jsonnetfile.lock.json

```diff
@@ -4,7 +4,7 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/brancz/kubernetes-grafana",
+          "remote": "https://github.com/brancz/kubernetes-grafana.git",
           "subdir": "grafana"
```

Comment:
is the git extension required? I am surprised to see it diff now or is this the result of some automation?

Reply:
Yeah no idea, I just ran the steps from https://github.com/openshift/cluster-monitoring-operator/pull/844#issue-446390661